libs/stache/src/lib/modules/action-buttons/action-buttons.component.ts

```diff
@@ -22,9 +22,8 @@ export class StacheActionButtonsComponent {
   }
 
   @Input()
-  @InputConverter(booleanConverter)
   public set showSearch(value: boolean | string | undefined) {
-    this.#_showSearch = value !== false;
+    this.#_showSearch = booleanConverter(value);
```

Comment:
Does this still default to `true`? Should it be `booleanConverter(value) !== false`?

Reply:
Fixed